app/models/concerns/errors_json.rb

```diff
@@ -3,8 +3,8 @@ module ErrorsJson
 
   included do
     fields do
-      errors_json :string, limit: 256, required: false
-      ignore_errors :boolean, default: false, required: true
+      errors_json :string, limit: 256, null: true
+      ignore_errors :boolean, default: false, null: false
```

Comment:
So you're going to migrate this column from `null: true` to `null: false`? I like cleaning it up like that... but I just wanted to confirm.

Reply:
`null: false` is the default with our most recent changes to `hobo_fields` (pre v3.0).  I can technically remove this `null: false` but I felt that it was good in this case to be verbose.  `required: true/false` isn't respected by hobo_fields for field declarations (only `belongs_to` relationships), so that was never doing anything here.

Reply:
Yes, I know the default swapped, for exactly this reason: 80% of the cases that defaulted to `null: true` should have been `null: false`. I was curious how far you were going to switch those over and migrate the schema?

Reading the hobo_fields source, I saw that they accept a separate `:required,` symbol in the arg array just after the `:boolean`. And also `:unique,`. They are documented here: https://github.com/Invoca/hobo_fields/blob/07046283e46f91901efc0ddae4716983ef2b52fb/test/api.rdoctest#L99-L98
Of course I want to swap them out for `validates: { presence: true }` later.

Reply:
Ah, you're right. Isn't it a bit odd to have this split of a field being `required` but allowing `null: true` at the database level? I feel like these two should be unified into a single attribute to avoid confusion in the future, or worse latent bugs.

Reply:
`required: false` was a no-op but it was using the default of `null: true` before, so that was the effect.

The way you have it now looks correct. `errors_json` can be `null`. `ignore_errors` cannot be `null`.